File: java/estruturas-repetitivas/Divisao.java
import java.util.Locale;
import java.util.Scanner;

public class Divisao {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double div;
        int x, i, num, den;

        System.out.print("Quantos casos voce vai digitar? ");
        x = sc.nextInt();

        for (i = 0; i < x; i++) {
           System.out.print("Entre com o numerador: ");
           num = sc.nextInt();
           System.out.print("Entre com o denominador: ");
           den = sc.nextInt(); 

           if (den == 0) {
            System.out.println("DIVISAO IMPOSSIVEL");
           } else {
            div = (double) num / den;
            System.out.printf("DIVISAO = %.2f %n", div);
           }
        }

        sc.close();
    }
}

File: js/estruturas-repetitivas/divisao.js
const readline = require('readline-sync');

const X = Number(readline.question("Quantos casos voce vai digitar? "));

for (let i = 0; i < X; i++) {
    let num = Number(readline.question("Entre com o numerador: "));
    let den = Number(readline.question("Entre com o denominador: "));

    if (den === 0) {
        console.log("DIVISAO IMPOSSIVEL");
    } else {
        let div = num / den;
        console.log(`DIVISAO = ${div.toFixed(2)}`);
    }
}